build.gradle

```diff
@@ -190,7 +190,7 @@ dependencies {
   compile group: 'uk.gov.hmcts.reform', name: 'logging-appinsights', version: versions.reformLogging
   compile group: 'uk.gov.hmcts.reform', name: 'service-auth-provider-client', version: '3.0.0'
 
-  compile group: 'uk.gov.hmcts.reform', name: 'sscs-common', version: '3.0.87'
+  compile group: 'uk.gov.hmcts.reform', name: 'sscs-common', version: '3.0.96-RC2'
```

Comment:
RC candidate used. Kind reminder to upgrade to the release version before merging this PR.